libs/elasticsearch/tests/unit_tests/_async/test_vectorstores.py

```diff
@@ -417,3 +417,92 @@ async def test_elasticsearch_hybrid_scores_guard(
             await hybrid_store.asimilarity_search_by_vector_with_relevance_scores(
                 [1, 2, 3]
             )
+
+    @pytest.mark.asyncio
+    async def test_parameter_forwarding_to_evectorstore(self) -> None:
+        """Test to catch missing AsyncEVectorStore parameters.
+
+        This test compares the AsyncEVectorStore constructor signature against what
+        AsyncElasticsearchStore actually forwards. If AsyncEVectorStore adds new
+        parameters, this test will fail and alert us to update AsyncElasticsearchStore.
+        """
+        import inspect
+        from unittest.mock import AsyncMock, patch
```

Comment:
The `AsyncMock` class is is already imported at the beginning of the file. Moreover, it's better to specify all the imports at the beginning of the file (see. [PEP 8](https://peps.python.org/pep-0008/)).